test/com/linkedin/drelephant/schedulers/OozieSchedulerTest.java

```diff
@@ -0,0 +1,182 @@
+package com.linkedin.drelephant.schedulers;
+
+import com.linkedin.drelephant.configurations.scheduler.SchedulerConfigurationData;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import mockit.Expectations;
+import mockit.Mocked;
+import org.junit.Test;
+import org.apache.oozie.client.OozieClient;
+import org.apache.oozie.client.OozieClientException;
+import org.apache.oozie.client.WorkflowJob;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class OozieSchedulerTest {
+  @Mocked
+  OozieClient oozieClient;
+
+  @Mocked
+  WorkflowJob workflowJob;
+
+  @Test
+  public void testOozieLoadInfoWithCompleteConf() throws Exception {
+    new Expectations() {{
+      workflowJob.getAppName();
+      result = "some-workflow-name";
+
+      oozieClient.getJobInfo("0004167-160629080632562-oozie-oozi-W");
+      result = workflowJob;
+    }};
```

Comment:
it's not easy to tell what's happening here when reading the first time.
maybe add a comment that `workflowJob` is mocked?